core/classes/Events/EventHandler.php

```diff
@@ -5,14 +5,21 @@
  * @package NamelessMC\Events
  * @author Samerton
  * @author Aberdeener
- * @version 2.0.0-pr13
+ * @version 2.1.0
  * @license MIT
  */
 class EventHandler {
 
     private static array $_events = [];
     private static array $_webhooks = [];
 
+    public static function discoverEvents(): void {
```

Comment:
Believe we should keep that modules self register the event to prevent outdated files, and if event require certain modules etc so better to keep so modules self register it

Reply:
Also with the autoDiscover er don't know if the event is internal or not anymore